airflow/www/views.py

```diff
@@ -515,11 +515,14 @@ def get_int_arg(value, default=0):
         state_color_mapping = State.state_color.copy()
         state_color_mapping["null"] = state_color_mapping.pop(None)
 
+        page_title = conf.get(section="webserver", key="site_title", fallback="DAGs")
```

Comment:
hmm.. same config with a different fallback?

Reply:
fallback only works if config is not provided, when config is provided `DAGs` won't be used

Reply:
> same config with a different fallback?

So amend the fallback? I think if the `site_title` config is set, then we use that; else, we default to the standard `DAGs` page title in the DAGs UI page.

Reply:
Maybe have 2 different configs, 1 for page_title and 1 for site_title unless this has already been discussed

Reply:
Yh I did ask about this earlier on, but I think based on what Kamil said we integrated the changes?

Reply:
I think we can use a more generic option name like `instance_name` which will be empty by default. When it is set, some strings will be replaced with the name of the instance, and in other cases, the description will be used more appropriately to the case. I don't see the benefit of splitting this into two options if the user will always set identical values two after. I don't see the benefit of splitting this into two options if the user always sets the same values in the two options. We really want to get it to be able to easily recognize different environments/instances of this application.

Reply:
Thanks both; OK so basically just rename `site_title` to `instance_name`? The default behaviour for the UI will be `DAGs` and the page title `Airflow`. I'll amend the code and docs.

Reply:
👍